CRM/Case/XMLProcessor/Report.php

```diff
@@ -738,6 +738,24 @@ private function redact($string, $printReport = FALSE, $replaceString = array())
    */
   public static function getCaseReport($clientID, $caseID, $activitySetName, $params, $form) {
 
+    $template = self::_getCaseReport($clientID, $caseID, $activitySetName, $params, $form);
+
+    // now run the template
+    $contents = $template->fetch('CRM/Case/XMLProcessor/Report.tpl');
+    return $contents;
+  }
+
+  /**
+   * @param int $clientID
+   * @param int $caseID
+   * @param string $activitySetName
+   * @param array $params
+   * @param CRM_Core_Form $form
+   *
+   * @return CRM_Core_Smarty
+   */
+  public static function _getCaseReport($clientID, $caseID, $activitySetName, $params, $form) {
```

Comment:
I think we are phasing out the convention of underscores for function names

Reply:
ok